docs/modules/data-structures/pages/map-config.adoc

```diff
@@ -23,8 +23,8 @@ ifdef::snapshot[]
 endif::[]
 
 ifndef::snapshot[]
-* link:https://github.com/hazelcast/hazelcast/blob/v{full-version}/hazelcast/src/main/resources/hazelcast-full-example.xml[hazelcast-full-example.xml]
-* link:https://github.com/hazelcast/hazelcast/blob/v{full-version}/hazelcast/src/main/resources/hazelcast-full-example.yaml[hazelcast-full-example.yaml]
+* link:https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-full-example.xml[hazelcast-full-example.xml]
```

Comment:
Sorry for interfering, but maybe a better approach would be to introduce a new variable e.g. `hazelcast-tag`, and use it here?
It seems that we have here a wrong usage of `full-version` variable. That variable means the version of the documentation but it was used here ad the version of the hazelcast (set in GH tag)

![image](https://github.com/user-attachments/assets/4e7a25f6-d5f6-4600-808d-1f186c724b93)

Reply:
> Sorry for interfering, but maybe a better approach would be to introduce a new variable e.g. `hazelcast-tag`, and use it here? It seems that we have here a wrong usage of `full-version` variable. That variable means the version of the documentation but it was used here ad the version of the hazelcast (set in GH tag)

@fantkolja already under discussion [here](https://github.com/hazelcast/hz-docs/pull/1355).

Reply:
And related PR: https://github.com/hazelcast/hz-docs/pull/1355